app/core/lifespan.py

```
@@ -147,3 +164,7 @@ async def lifespan(app: FastAPI):
             ip_stop.set()
         except Exception:
             pass
+        try:
+            megakino_stop.set()
+        except Exception:
+            pass
```

Ruff (0.14.10)

`try`-`except`-`pass` detected, consider logging the exception

(S110)